test/run-k8s-integration-ci.sh

```diff
@@ -29,7 +29,7 @@ readonly run_intree_plugin_tests=${RUN_INTREE_PLUGIN_TESTS:-false}
 readonly use_kubetest2=${USE_KUBETEST2:-true}
 readonly test_pd_labels=${TEST_PD_LABELS:-true}
 readonly migration_test=${MIGRATION_TEST:-false}
-readonly test_disk_image_snapshot=${TEST_DISK_IMAGE_SNAPSHOT:-false}
+readonly test_disk_image_snapshot=${TEST_DISK_IMAGE_SNAPSHOT:-true}
```

Comment:
Can we use multiple snapshot class files with the test flag? If so, we should just set the CI to run both the conventional & disk image snapshots.